Library/Homebrew/caveats.rb

```diff
@@ -46,6 +47,14 @@ def zsh_completion_caveats
     end
   end
 
+  def fish_completion_caveats
+    if keg and keg.completion_installed? :fish then <<-EOS.undent
+      fish completion has been installed to:
+        #{HOMEBREW_PREFIX}/etc/fish/completions
+      EOS
```

Comment:
Not sure what I think about this considering the relatively niche popularity of fish. Maybe worth sniffing for what the user's shell is?